packages/elements/src/components/ino-sidebar/readme.md

```diff
@@ -92,13 +92,6 @@ https://github.com/material-components/material-components-web/tree/master/packa
 | `"header"`  | For the header element |
 
 
-## CSS Custom Properties
```

Comment:
This table seems to be disappeared by accident

Reply:
Yeah this was unintentional...